CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# [0.12.3](https://github.com/dashevo/mn-bootstrap/compare/v0.12.2...v0.12.3) (2020-04-28)
```

Comment:
0.12.3 is already released :)

Reply:
I know! I don't know why this happened or how to fix it :( I think I need to rebase or merge something from upstream or another branch?

Reply:
You just need to pull master from upstream and merge it to this branch.